src/ContainerHealthLogs.jsx

```diff
@@ -0,0 +1,117 @@
+                    <DescriptionList isAutoFit className='container-details-healthcheck'>
+                        <DescriptionListGroup>
+                            <DescriptionListTerm>{_("Command")}</DescriptionListTerm>
+                            <DescriptionListDescription>{utils.quote_cmdline(healthCheck.Test)}</DescriptionListDescription>
+                        </DescriptionListGroup>
+                        {healthCheck.Interval && <DescriptionListGroup>
+                            <DescriptionListTerm>{_("Interval")}</DescriptionListTerm>
+                            <DescriptionListDescription>{format_nanoseconds(healthCheck.Interval)}</DescriptionListDescription>
+                        </DescriptionListGroup>}
+                        {healthCheck.Retries && <DescriptionListGroup>
+                            <DescriptionListTerm>{_("Retries")}</DescriptionListTerm>
+                            <DescriptionListDescription>{healthCheck.Retries}</DescriptionListDescription>
+                        </DescriptionListGroup>}
+                        {healthCheck.StartPeriod && <DescriptionListGroup>
+                            <DescriptionListTerm>{_("Start period")}</DescriptionListTerm>
+                            <DescriptionListDescription>{format_nanoseconds(healthCheck.StartPeriod)}</DescriptionListDescription>
+                        </DescriptionListGroup>}
+                        {healthCheck.Timeout && <DescriptionListGroup>
+                            <DescriptionListTerm>{_("Timeout")}</DescriptionListTerm>
+                            <DescriptionListDescription>{format_nanoseconds(healthCheck.Timeout)}</DescriptionListDescription>
+                        </DescriptionListGroup>}
+                        {failingStreak !== 0 && <DescriptionListGroup>
+                            <DescriptionListTerm>{_("Failing streak")}</DescriptionListTerm>
+                            <DescriptionListDescription>{failingStreak}</DescriptionListDescription>
+                        </DescriptionListGroup>}
+                    </DescriptionList>
+                </FlexItem>
+                <FlexItem>
+                    <Button variant="secondary" onClick={() => {
+                        client.runHealthcheck(container.isSystem, container.Id)
+                                .catch(ex => {
+                                    const error = cockpit.format(_("Failed to run health check on container $0"), container.Names);
+                                    onAddNotification({ type: 'danger', error, errorDetail: ex.message });
+                                });
+                    }}>
+                        {_("Run health check")}
+                    </Button>
+                </FlexItem>
+            </Flex>
+            <ListingTable aria-label={_("Logs")}
+                          variant='compact'
+                          columns={['Last 5 runs', _("Started at")]}
```

Comment:
`Last 5 Runs` should be translated as well right?

Reply:
Indeed!